pages/firstjump.html

```diff
@@ -49,40 +53,47 @@ <h1>Physical requirements:</h1>
         <h1>Length of Jump:</h1>
         <p>
           Most dives last approximately 2 hours, which includes paperwork,
-          training, equipment preparation, aircraft ride,<br />
-          the jump itself, and removal of equipment. We cannot guarantee flight
-          conditions or specify exact times for the <br />day of your jump. Be
+          training, equipment preparation, aircraft ride, the jump itself, and removal of equipment. We cannot guarantee flight
+          conditions or specify exact times for the day of your jump. Be
           prepared to wait up to 4 hours if needed.
         </p>
       </div>
     </div>
+  </div>
+
     <div id="formSection">
+      <div>
       <h1 id="formHeader">
         Set up an appointment <br />
         with one of our specialists to get started:
       </h1>
+    </div>
       <div id="formContainer">
-        <div id="form">
+        <form id="form">
           <div>
             <label for="name">Name:*</label>
-            <input type="text" id="name" />
+            <input type="text" id="name" name="Name"/>
```

Comment:
Lets stick with lowercase snake case for the names. You mix and match a little with the capitals